lib.rs

```diff
@@ -251,31 +251,22 @@ impl slog::Drain for StdLog {
 
         let lazy = LazyLogString::new(info, logger_values);
         /*
-         * TODO: Support `log` crate key_values here.
-         *
-         * This requires the log/kv_unstable feature here.
-         *
-         * Not supporting this feature is backwards compatible
-         * and it shouldn't break anything (because we've never had),
-         * but is undesirable from a feature-completeness point of view.
-         *
-         * However, this is most likely not as powerful as slog's own
-         * notion of key/value pairs, so I would humbly suggest using `slog`
-         * directly if this feature is important to you ;)
-         *
          * This avoids using the private log::__private_api_log api function,
          * which is just a thin wrapper around a `RecordBuilder`.
          */
-        log::logger().log(
-            &log::Record::builder()
-                .args(format_args!("{}", lazy))
```

Comment:
We are, on line 269. 

I had to move it there, because I conditionally (at compile time) call a `key_values` on the record_build, but due to https://github.com/rust-lang/rust/issues/92698, the call to `format_args!` has to be in the same statement that we call `build()` in.

I can add a comment explaining that.